### PR TITLE
Release v0.9.0

### PR DESCRIPTION
**Pull request by waltjones**
## Description of the change

Release v0.9.0 version update.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Maintenance
- [x] New release

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers assigned
- [ ] Issue from task tracker has a link to this pull request
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

**Comment by calvinf** [comment]
Not that you need my approval, but LGTM and looking forward to getting the TypeScript definitions.

**Comment by waltjones** [comment]
The project is set up that there needs to be one approver in the Rollbar org. I can approve and merge other folks, but not myself.

@pawelsz-rb should be able to help us out.

**Review by pawelsz-rb — APPROVED** [review]

**Pull request merged**
